icpz.h

```diff
@@ -11,6 +11,46 @@
 
 static uint8_t CRC_BiSS_43_30bit (uint32_t w_InputData);
 
+#define ICPZ_SET_DEBUG_VARIABLES(prefix, api, icpz) \
+    api.add_api_variable(prefix "ai_phase", new const APICallbackFloat([](){ return icpz.get_ai_phase(); }));\
+    api.add_api_variable(prefix "ai_scale", new const APICallbackFloat([](){ return icpz.get_ai_scale(); }));\
```

Comment:
Just curious, why do this?

Reply:
The api is due for some kind of refactor. This was the quickest way I could think of to maintain the existing strings but have them defined in rom. Most other strings are fixed, but for this PZ I was doing "mai_phase" and "oai_phase" for motor and output respectively.